### ISSUE TITLE
Wrong UI language: Accept-Language setting is read in the wrong order

### ISSUE THREAD

**Issue by jval** [EFForg/privacybadgerfirefox#660]
Added multiple languages to the preferred content language setting in Firefox (which is sent as Accept-Language header to the websites).

Privacy Badger changed its UI language to the language which has the smallest priority. Naturally it should set the UI language to the one with the highest priority.

**Comment by cooperq** [comment]
I don't think that we control the order that languages are read in, I suspect that this is a bug in firefox and not in privacy badger.

**Comment by Seegras** [comment]
Since this is the only add-on with this problem, chances are the bug is with privacy badger. Besides, why would you even use Accept-Language? You could use the locale of the browser. 

Please reopen and investigate.

**Comment by cooperq** [comment]
A fair point, this is at least worth investigating.

**Comment by S-Graham** [comment]
The priority ordering of languages in Firefox has no effect on Privacy Badger. If I have Italian in any position, PB uses that. If I delete Italian, it uses German, again wherever it occurs in the list. There seems no way that I can get PB to use English unless it's the only entry.

As the previous poster pointed out, actually this is an incorrect design decision anyway. The Privacy Badger UI should use the language of the browser UI, i.e. the locale.

**Comment by cooperq** [comment]
this should be fixed by the move to web extensions.

**Comment by HayoBaan** [comment]
Just so this one doesn't get forgotten; I too see the wrong UI language selected. A minor thing (I do speak the selected language), but annoying still 😉

**Comment by cooperq** [comment]
I would happily accept a patch to fix this, but I don't personally have time to work on it. I am currently moving full speed ahead on the transition to the web extensions API.

**Comment by augusto-herrmann** [comment]
Just to let you know, I have just installed Privacy Badger for the first time ever, after having just learned of its existence, and was surprised to see the same problem as reported here.
